src/mlpack/methods/ann/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(SOURCES
   rnn_impl.hpp
   brnn.hpp
   brnn_impl.hpp
+  layer_names.hpp
```

Comment:
Can you write a simple test for the visitor? Similar to https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/ann_visitor_test.cpp

Reply:
Sure, I'll get it done tomorrow, pushing the changes here for now